### DISCUSSION TITLE
use more than one name space at once

### DISCUSSION THREAD

**Discussion by rubenmarcus** [amannn/next-intl#133] [Q&A]
is it possible to use more than one namespace at my component?

I dont see how if I need to load like : 
  const {t} = useTranslation(['Home', ' Buttons'])
  
  ts complains...
how to do that?

thanks

**Comment by amannn** [comment]
Hi @rubenmarcus,

this feature is intentionally not supported currently (see https://github.com/amannn/next-intl/issues/41).

You can solve it like this though: 

```js
const t = useTranslations();
t('Home.title');
t('Butons.title');
```

Hope this helps!

**Reply by rubenmarcus** [reply]
@amannn  thanks!
